docs/standard/datetime/snippets/timeprovider-overview/csharp/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿using ExampleProject;
+
+Console.WriteLine(@"_./-==--======- getlocal -======--==-\._");
+
+//<GetLocal>
+Console.WriteLine($"Local: {TimeProvider.System.GetLocalNow()}");
+Console.WriteLine($"Utc:   {TimeProvider.System.GetUtcNow()}");
+
+/* This example produces output similar to the following:
+ *
+ * Local: 12/5/2024 10:41:14 AM -08:00
+ * Utc:   12/5/2024 6:41:14 PM +00:00
+*/
+//</GetLocal>
+
+Console.WriteLine(@"_./-==--======- timestamp -======--==-\._");
+
+//<Timestamp>
+long stampStart = TimeProvider.System.GetTimestamp();
+Console.WriteLine($"Starting timestamp: {stampStart}");
+
+long stampEnd = TimeProvider.System.GetTimestamp();
```

Comment:
>stampEnd [](http://example.com/codeflow?start=5&length=8)

Would it make sense to insert like Thread.Sleep(1000) before this line?

Reply:
I think the current code really demonstrates that the timer is indeed high resolution.